src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2503,6 +2503,26 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         return flags;
     }
 
+    til::point TermControl::_toControlOrigin(const til::point& terminalPos)
+    {
+        const til::size fontSize{ til::math::flooring, CharacterDimensions() };
+
+        // Convert text buffer cursor position to client coordinate position
+        // within the window. This point is in _pixels_
+        const til::point clientCursorPos{ terminalPos * fontSize };
+
+        // Get scale factor for view
+        const double scaleFactor = DisplayInformation::GetForCurrentView().RawPixelsPerViewPixel();

```

Comment:
hmm, does TC not already know this?

Reply:
This _might_ also be `SwapChainPanel().CompositionScaleX()`...